### ISSUE TITLE
Can't install in a clean environment

### ISSUE THREAD

**Issue by viklund** [EGA-archive/crypt4gh#7]
## Problem

When installing the library I get the following error:

```Collecting git+https://github.com/EGA-archive/crypt4gh.git
  Cloning https://github.com/EGA-archive/crypt4gh.git to /private/var/folders/1d/71z0tzqj39s7m3_yq504_t1r0000gn/T/pip-req-build-prc_7eza
  Running command git clone -q https://github.com/EGA-archive/crypt4gh.git /private/var/folders/1d/71z0tzqj39s7m3_yq504_t1r0000gn/T/pip-req-build-prc_7eza
    ERROR: Command errored out with exit status 1:
     command: /private/tmp/venv/bin/python3.8 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/1d/71z0tzqj39s7m3_yq504_t1r0000gn/T/pip-req-build-prc_7eza/setup.py'"'"'; __file__='"'"'/private/var/folders/1d/71z0tzqj39s7m3_yq504_t1r0000gn/T/pip-req-build-prc_7eza/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/1d/71z0tzqj39s7m3_yq504_t1r0000gn/T/pip-req-build-prc_7eza/pip-egg-info
         cwd: /private/var/folders/1d/71z0tzqj39s7m3_yq504_t1r0000gn/T/pip-req-build-prc_7eza/
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/1d/71z0tzqj39s7m3_yq504_t1r0000gn/T/pip-req-build-prc_7eza/setup.py", line 7, in <module>
        from crypt4gh import __version__, __author__, __title__, __doc__ as lega_doc, __license__, PROG
      File "/private/var/folders/1d/71z0tzqj39s7m3_yq504_t1r0000gn/T/pip-req-build-prc_7eza/crypt4gh/__init__.py", line 25, in <module>
        from nacl.exceptions import InvalidkeyError, BadSignatureError, CryptoError
    ModuleNotFoundError: No module named 'nacl'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

## Steps to reproduce

```bash
virtualenv /tmp/venv
source /tmp/venv/bin/activate
pip install git+https://github.com/EGA-archive/crypt4gh.git
```

## Probable cause

`setup.py` imports the ga4gh library to get the `__version__` strings and some other metadata. But `ga4gh/__init__.py` imports both `nacl` and `cryptography`, both of which aren't installed yet.

**Comment by dtitov** [comment]
+1, I have the same.

**Comment by silverdaz** [comment]
Fixed by commit #64302b8749
